lib/cc/pull_requests.rb

```diff
@@ -2,7 +2,28 @@ class CC::PullRequests < CC::Service
   def receive_test
     setup_http
 
-    receive_test_status
+    response = receive_test_status
+
+    if welcome_comment_implemented? && config.welcome_comment_enabled
+      welcome_comment_response = receive_test_welcome_comment
+      if response[:ok] && welcome_comment_response[:ok]
+        {
+          ok: true,
+          message: "Access token is valid",
```

Comment:
At this point, we haven't posted the comment or updated the status yet, but verified that with the provided token, we'll be able to (this is for the "Test Service" button). Given, that I think the wording makes sense but am able to be convinced otherwise.

I think this code is kind of confusing currently... going to spend some time trying to make it more straight-forward now